src/client/components/app/SliderCard/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { ReactNode, useState } from 'react';
+import styled from 'styled-components/macro';
+
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  CardSizeType,
+  Button,
+  Icon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+} from '@components/common';
+
+const ArrowIcon = styled(Icon)`
+  fill: currentColor;
+  width: 1rem;
+`;
+
+const ArrowButton = styled(Button)`
+  padding: 0.7rem;
+`;
+
+const SliderCounter = styled.div`
+  display: flex;
+  align-items: center;
+  position: absolute;
+  bottom: ${({ theme }) => theme.layoutPadding};
+  right: ${({ theme }) => theme.layoutPadding};
+  font-weight: 700;
+  font-size: 1.6rem;
+
+  ${ArrowButton}:not(:first-child) {
+    margin-left: 0.8rem;
+  }
+  ${ArrowButton}:not(:last-child) {
+    margin-right: 0.8rem;
+  }
+`;
+
+const StyledCardContent = styled(CardContent)`
+  margin: ${({ theme }) => theme.card.padding};
+  margin-bottom: 0;
+  font-size: 1.6rem;
+  line-height: 2.4rem;
+  color: inherit;
+`;
+
+const StyledCard = styled(Card)`
+  // max-width: max-content;
```

Comment:
can. we remove this comment?

Reply:
Fixed in aef61c0699b977d420319fb3f99856c4aa59fcdd